modules/button/render.html.twig

```diff
@@ -2,5 +2,5 @@
     <a href="{{ sfs_cms_url(button_link) }}"
             {{ sfs_cms_route_attr(button_link)}}
             id="{{ id|default('') }}"
-            class="{{ button_style|default(button_style_custom|default('btn btn-primary')) }}">{{ button_text|sfs_cms_trans|raw }}</a>
+            class="{{ button_style|default('btn btn-primary') }} {{ button_style_custom|default('') }}">{{ button_text|sfs_cms_trans|raw }}</a>
```

Comment:
This will render **class="custom button-style-custom-classes"** or if it's empty **class="btn btn-primary** (because of default value) **button-style-custom-classes""**.

The solution depends on the way the button_style "custom" status is stored.

Reply:
Done!